examples/eco_force/Ni.eam.fs

```diff
@@ -0,0 +1,6006 @@
+Sourse: The potential parameters were provided by Prof. G.J. Ackland (g.j.ackland@ed.ac.uk). Please, contact him if you have any questions.
```

Comment:
please do not add copies of potential files, if it can be avoided. rather use a (relative) symbolic link to a potential already present in the potentials folder. similar to other examples.